### ISSUE TITLE
[1.10.2] Game crashes on load

### ISSUE THREAD

**Issue by BLAMM67** [Shinoow/AbyssalCraft#279]
#### Description:

I was checking out the latest version of Abyssalcraft in a single-player game with a custom modpack. Don't think its related, but at the time I was trying out the Create a Spell page in the Abyssalnomicon. The game crashed and now crashes every time when I try loading that world again. The crash log is always the same, a null pointer exception during EntityLesserShoggoth.sprayAcid().

#### Crash report:

[Crash report](https://pastebin.com/pHDJrBP9)

----

#### Affected versions ("latest" is **NOT** a version):
- Minecraft: 1.10.2
- Forge: 12.18.3.2511
- AbyssalCraft: 1.9.4.7

#### Latest log file for when the issue was present:

[Log File](https://pastebin.com/4PkaaRWF)

**Comment by CurePindal** [comment]
Same issue with the ticking entity from com.shinoow.abyssalcraft.common.entity.EntityLesserShoggoth.sprayAcid

Crashlog: https://pastebin.com/V7Sd54s4

Error in latest.log:
```
[14:55:40] [Server thread/ERROR]: Encountered an unexpected exception
net.minecraft.util.ReportedException: Ticking entity
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:715) ~[MinecraftServer.class:?]
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:613) ~[MinecraftServer.class:?]
	at net.minecraft.server.integrated.IntegratedServer.func_71217_p(IntegratedServer.java:240) ~[bzl.class:?]
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:471) [MinecraftServer.class:?]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_101]
Caused by: java.lang.NullPointerException
	at com.shinoow.abyssalcraft.common.entity.EntityLesserShoggoth.sprayAcid(EntityLesserShoggoth.java:458) ~[EntityLesserShoggoth.class:?]
	at com.shinoow.abyssalcraft.common.entity.ai.EntityAILesserShoggothAttackMelee.checkAndPerformAttack(EntityAILesserShoggothAttackMelee.java:189) ~[EntityAILesserShoggothAttackMelee.class:?]
	at com.shinoow.abyssalcraft.common.entity.ai.EntityAILesserShoggothAttackMelee.func_75246_d(EntityAILesserShoggothAttackMelee.java:173) ~[EntityAILesserShoggothAttackMelee.class:?]
	at net.minecraft.entity.ai.EntityAITasks.func_75774_a(SourceFile:129) ~[tr.class:?]
	at net.minecraft.entity.EntityLiving.func_70626_be(EntityLiving.java:900) ~[sg.class:?]
	at net.minecraft.entity.EntityLivingBase.func_70636_d(EntityLivingBase.java:2183) ~[sf.class:?]
	at net.minecraft.entity.EntityLiving.func_70636_d(EntityLiving.java:685) ~[sg.class:?]
	at net.minecraft.entity.monster.EntityMob.func_70636_d(SourceFile:47) ~[yx.class:?]
	at com.shinoow.abyssalcraft.common.entity.EntityLesserShoggoth.func_70636_d(EntityLesserShoggoth.java:224) ~[EntityLesserShoggoth.class:?]
	at net.minecraft.entity.EntityLivingBase.func_70071_h_(EntityLivingBase.java:2013) ~[sf.class:?]
	at net.minecraft.entity.EntityLiving.func_70071_h_(EntityLiving.java:384) ~[sg.class:?]
	at net.minecraft.entity.monster.EntityMob.func_70071_h_(SourceFile:52) ~[yx.class:?]
	at com.shinoow.abyssalcraft.common.entity.EntityLesserShoggoth.func_70071_h_(EntityLesserShoggoth.java:137) ~[EntityLesserShoggoth.class:?]
	at net.minecraft.world.World.func_72866_a(World.java:1964) ~[aid.class:?]
	at net.minecraft.world.WorldServer.func_72866_a(WorldServer.java:839) ~[ls.class:?]
	at net.minecraft.world.World.func_72870_g(World.java:1934) ~[aid.class:?]
	at net.minecraft.world.World.func_72939_s(World.java:1750) ~[aid.class:?]
	at net.minecraft.world.WorldServer.func_72939_s(WorldServer.java:620) ~[ls.class:?]
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:709) ~[MinecraftServer.class:?]
	... 4 more
```

Can no longer load the world. Instacrash from the ticking entity.

**Comment by Shinoow** [comment]
Seems like I missed a null check when downporting that code to 1.10.2. I'll see about putting an update together today to address that (already got some other minor things that could warrant an update).

**Comment by Shinoow** [comment]
Fixed in 1.9.4.8
